packages/smart-wallet/src/smartWallet.js

```diff
@@ -1052,7 +1052,7 @@ export const prepareSmartWallet = (baggage, shared) => {
         /**
          * Umarshals the actionCapData and delegates to the appropriate action handler.
          *
-         * @param {import('@endo/marshal').CapData<string>} actionCapData of type BridgeAction
+         * @param {import('@endo/marshal').CapData<string | null>} actionCapData of type BridgeAction
```

Comment:
@dckc please confirm

Reply:
yes, it looks like `actionCapData` is sent to the board marshaller, where `slotToVal` handles `null`.